### ISSUE TITLE
Validation set and early stopping (TimeGrad)

### ISSUE THREAD

**Issue by victorsergio** [zalandoresearch/pytorch-ts#137]
Hi, I would like to know if the TimeGrad estimator already implement the option of using a validation set?

How can I implement an early stopping mechanism using the TimeGrad estimator?

Thanks in advance.
